egs/musdb18/OpenUnmix/test_dataloader.py

```diff
@@ -0,0 +1,134 @@
+from asteroid.data import MUSDB18Dataset
+import argparse
+import torch
+from pathlib import Path
+import tqdm
+
+
+class Compose(object):
```

Comment:
I've been thinking about adding this for a while.. We'll probably move it inside asteroid.

Reply:
yes, please